cmd/microcloud/ask.go

```diff
@@ -565,7 +647,7 @@ func (c *CmdControl) askRemotePool(systems map[string]InitSystem, autoSetup bool
 	return nil
 }
 
-func (c *CmdControl) askNetwork(sh *service.Handler, systems map[string]InitSystem, autoSetup bool) error {
```

Comment:
Have you tested if the modifications to the network questions will correctly be skipped if MicroCeph is not installed?

Reply:
yes